{"name": "virtualenv", "cve_id": "CVE-2018-17793", "publish_date": "2018-09-30T19:29Z", "cwe_id": "CWE-254", "owner": "virtualenv", "affected_versions": [{"version_value": "16.0.0", "operator": "="}], "references": ["https://github.com/pypa/virtualenv/issues/1207", "https://www.exploit-db.com/exploits/45528/"], "description": "** DISPUTED ** Virtualenv 16.0.0 allows a sandbox escape via \"python $(bash >&2)\" and \"python $(rbash >&2)\" commands. NOTE: the software maintainer disputes this because the Python interpreter in a virtualenv is supposed to be able to execute arbitrary code.", "latest_version": "16.7.8", "language": "python", "license": "MIT", "repository_summary": "Virtual Python Environment builder http://virtualenv.pypa.io", "repo_url": "https://github.com/pypa/virtualenv", "impact": {"cvss2_score": 10.0, "cvss2_severity": "HIGH", "vector_string": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:C/C:H/I:H/A:H", "av": "NETWORK", "ac": "LOW", "pr": "NONE", "c": "HIGH", "i": "HIGH", "a": "HIGH", "impact_score": 6.0, "exploitability_score": 3.9, "cvss3_score": 10.0, "cvss3_severity": "CRITICAL", "ui": "NONE", "s": "CHANGED"}, "ochrona_severity_score": 10.0}